fictadvisor-api/src/v2/utils/documentation/date/index.ts

```diff
@@ -0,0 +1,7 @@
+import { DateDocumentationSemesters } from './semesters';
+import {DateDocumentationCurrentSemester} from "./CurrentSemester";
+
+export const DateDocumentation = {
+  SEMESTERS: DateDocumentationSemesters,
+  CURRENT_SEMESTER: DateDocumentationCurrentSemester,
+};
```

Comment:
names of Documentation properties must be the same as corresponding methods of controller `GET_PREVIOUS_SEMESTERS` and `GET_CURRENT_SEMESTER`